classes/local/store/digitalocean/client.php

```diff
@@ -35,9 +35,16 @@ public function __construct($config) {
         $this->autoloader = $CFG->dirroot . '/local/aws/sdk/aws-autoloader.php';
         $this->testdelete = false;
 
+
```

Comment:
Can you please remove this blank line (breaks code standards ci) and this this should be good to land

Reply:
Done.